db/chi_cache.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE `chi_cache` (
+  `key` VARCHAR(767),
+  `value` LONGTEXT,
+  PRIMARY KEY (`key`)
```

Comment:
Shouldn't we also add a `created` column that would be set to the creation date of the entry? It could be useful to remove old entries automatically, in case the cache gets dirty over time.